include/SANDDigitizationEDEPSIM.h

```diff
@@ -48,12 +48,34 @@ void group_hits_by_tube(TG4Event* ev, const SANDGeoManager& geo,
 
 void create_digits_from_hits(const SANDGeoManager& geo,
                              std::map<int, std::vector<hit> >& hits2Tube,
-                             std::vector<dg_tube>& digit_vec);
+                             std::vector<dg_wire>& digit_vec);
 
 void digitize_stt(TG4Event* ev, const SANDGeoManager& geo,
-                  std::vector<dg_tube>& digit_vec);
+                  std::vector<dg_wire>& digit_vec);
 }  // namespace stt
 
+namespace chamber
+{
+void group_hits_by_wire(TG4Event* ev, const SANDGeoManager& geo,
+                        std::map<int, std::vector<hit> >& hits2wire);
+
+bool isInWire(SANDWireInfo& wire, TVector3& point);
```

Comment:
From the function name, it seems that neither the wire nor the point will be modified.

Anything more complex than an `int `or a `double `should be passed as a `const&` if it is not an IN-OUT argument.

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f16-for-in-parameters-pass-cheaply-copied-types-by-value-and-others-by-reference-to-const